source/_docs/disaster-recovery.md

```diff
@@ -31,5 +31,62 @@ Pantheon’s disaster response team will proactively notify you of a zone failur
 ## Limitations
 Redis and Solr are not currently supported on DR-enabled sites.
```

Comment:
## Limitations
Redis and Solr are not currently supported on DR-enabled sites.  For more information see our documentation on how to [safely remove Redis](https://pantheon.io/docs/redis/#safely-remove-redis) and how to [safely remove Solr](https://pantheon.io/docs/solr-drupal-8/#safely-remove-solr).